lib/beaker/dsl/install_utils/puppet_utils.rb

```diff
@@ -545,6 +545,11 @@ def install_puppetlabs_release_repo ( host )
         #
         # @note This method only works on redhat-like and debian-like hosts.
         #
+        # @note You can pass in additional package_repos to try using the global
+        #   options hash with the +:dev_builds_repos+ key.  This can be a
+        #   string value, or an array of repos to look through.  It will look
+        #   through them in the order given.
+        #
         def install_puppetlabs_dev_repo ( host, package_name, build_version,
                                   repo_configs_dir = 'tmp/repo_configs' )
```

Comment:
Let's pass in the options here with opts = options instead of using the global options directly in the method.